src/mail/send-message.js

```diff
@@ -0,0 +1,86 @@
+const nodemailer = require('nodemailer');
```

Comment:
Will we be using nodemailer for production?

Reply:
For now, that's the plan; I'm working on a branch trying to use `sendmail` so that we don't need to connect to an SMTP server at all, but having some trouble with that.